java/junit5/src/test/java/com/saucelabs/saucebindings/junit5/examples/LocalExample.java

```diff
@@ -0,0 +1,87 @@
+package com.saucelabs.saucebindings.junit5.examples;
+
+import com.saucelabs.saucebindings.SaucePlatform;
+import com.saucelabs.saucebindings.SauceSession;
+import com.saucelabs.saucebindings.junit5.SauceBindingsExtension;
+import com.saucelabs.saucebindings.options.SauceOptions;
+import java.time.Duration;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.junit.jupiter.api.extension.TestWatcher;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class LocalExample {
+  WebDriver driver;
+  SauceSession session;
+
+  @RegisterExtension
+  static SauceBindingsExtension sauceExtension = new SauceBindingsExtension(getSauceOptions());
+
+  @RegisterExtension TestWatcher testWatcher = new LocalTestWatcher();
+
+  @BeforeAll
+  public static void disableSauce() {
```

Comment:
@titusfortner I don't understand why we are disabling Sauce in a `beforeAll`?

Reply:
To show how to run locally. Running on sauce is default. Need to disable to run local.